src/NzbDrone.Core/Update/UpdateCheckService.cs

```diff
@@ -31,7 +33,13 @@ public class CheckUpdateService : ICheckUpdateService
 
         public UpdatePackage AvailableUpdate()
         {
-            return _updatePackageProvider.GetLatestUpdate(_configFileProvider.Branch, BuildInfo.Version);
+            _latestUpdate = _updatePackageProvider.GetLatestUpdate(_configFileProvider.Branch, BuildInfo.Version);
+            return _latestUpdate;
+        }
+
+        public UpdatePackage LatestUpdate()
```

Comment:
This is only required because of the approach being used, I don't think its needed.